src/main/java/com/google/api/codegen/transformer/go/GoGapicSurfaceTestTransformer.java

```diff
@@ -68,13 +81,50 @@ private MockCombinedView generateMockServiceView(
               .grpcMethods(mockServiceTransformer.createMockGrpcMethodViews(context))
               .build());
     }
+    for (Interface service : new InterfaceView().getElementIterable(model)) {
+      // We don't need any import here.
+      SurfaceTransformerContext context =
+          SurfaceTransformerContext.create(
+              service,
+              apiConfig,
+              GoGapicSurfaceTransformer.createTypeTable(),
+              namer,
+              featureConfig);
+      tests.add(
+          GapicSurfaceTestClassView.newBuilder()
+              .apiSettingsClassName("")
```

Comment:
This is okay but not very readable to pass empty strings here. If the field is not used by Go, we can mark it nullable in the view class or we can have some constants such as UNUSED